$OpenBSD: patch-configure_ac,v 1.5 2019/01/01 21:46:15 fcambus Exp $

Don't build a redundant copy of the documentation.
Don't run fc-cache on install.
Avoid picking up net/libnet.

Index: configure.ac
--- configure.ac.orig
+++ configure.ac
@@ -1260,12 +1260,7 @@ fi
 AC_CHECK_PROGS(MAKEINFO, makeinfo, no)
 
 AC_CHECK_PROGS(TEXI2DVI, texi2dvi, no)
-AC_CHECK_PROGS(PDFTEX, pdftex, no)
-if test x"$TEXI2DVI" = "xno" -o x"$PDFTEX" = "xno"; then
-  AM_CONDITIONAL(BUILD_PDF, false)
-else
-  AM_CONDITIONAL(BUILD_PDF, true)
-fi
+AM_CONDITIONAL(BUILD_PDF, false)
 
 AC_CHECK_PROGS(DOS2UNIX, dos2unix, dos2unix)
 
@@ -2370,8 +2365,6 @@ if test x"$is_unix_x11" = "xyes" -o x"$is_unix_macosx"
                                          LIBS="-lsocket $LIBS"],,)
     AC_CHECK_LIB(bsd, gethostbyname,[ NETPLAY_LIBS="-lbsd $NETPLAY_LIBS";
                                       LIBS="-lbsd $LIBS"],,)
-    AC_CHECK_LIB(net, gethostbyname,[ NETPLAY_LIBS="-lnet $NETPLAY_LIBS";
-                                      LIBS="-lnet $LIBS"],,)
     AC_CHECK_LIB(inet, gethostbyname,[ NETPLAY_LIBS="-linet $NETPLAY_LIBS";
                                        LIBS="-linet $LIBS"],,)
 
@@ -3012,12 +3005,7 @@ if test x"$enable_lame" != "xno" ; then
   fi
 fi
 
-AC_PATH_PROGS(FCCACHE, fc-cache, false)
-if test x"$FCCACHE" = "xfalse"; then
-  AM_CONDITIONAL(HAVE_FC_CACHE, false)
-else
-  AM_CONDITIONAL(HAVE_FC_CACHE, true)
-fi
+AM_CONDITIONAL(HAVE_FC_CACHE, false)
 
 if test x"$is_amigaos4" = "xyes"; then
   dnl for AmigaOS4 native resid needs to be configured without dependency tracking
